Gems/Multiplayer/Code/CMakeLists.txt

```diff
@@ -154,7 +114,8 @@ if (PAL_TRAIT_BUILD_HOST_TOOLS)
                 Gem::Multiplayer.Static
                 Gem::Multiplayer.Builders
     )
-
+    
+    ly_create_alias(NAME Multiplayer.Builders  NAMESPACE Gem TARGETS Gem::Multiplayer.Editor)
     # use the Multiplayer.Editor module in tools like the Editor:  Such tools also get the visual debug view:
     ly_create_alias(NAME Multiplayer.Tools     NAMESPACE Gem TARGETS Gem::Multiplayer.Editor Gem::Multiplayer.Debug Gem::Multiplayer.Builders)
```

Comment:
Now that the `Multiplayer.Builders` variant is aliasing the `Gem::Multiplayer.Editor` target, the `Multiplayer.Tools` variant can remove `Gem::Multiplayer.Editor` target from its list of targets

Reply:
Done.